### DISCUSSION TITLE
Mixed messages about 2FA

### DISCUSSION THREAD

**Discussion by bearbones** [community/community#113695] [Other Feature Feedback, Questions, & Ideas]
I already have two 2FA methods, SMS and GitHub app, but my account page has a message that I must "update" my 2FA methods or else my account will be limited. The FAQ page about 2FA doesn't say that I'm required to use something other than SMS and the GitHub app for 2FA, but it doesn't directly contradict the message either.

Therefore, I feel like I'm being told that my account will be limited unless I get a new phone number. I don't know if this is an oversight in the copywriting or in the active logic of the policy.

**Comment by hpsin** [comment]
Thanks @bearbones ,can you share a screenshot of this? It doesn't line up with anything we intend to happen, from your description, so it'll help us find what sounds like a bug (or explain it better).

There is a 28 day checkup after you've set up 2fa, that makes sure you've got it working. It only shows if you haven't done 2fa since you did setup. Maybe that's it? That should let you go through if you can successfully do 2fa,and only offer reconfiguration if you're unable to do 2fa.

**Reply by bearbones** [reply]
Yes, here is the notice and the panel with my 2FA methods 

![Screenshot_20240322-105930.png](https://github.com/community/community/assets/57124166/f22464ea-2d8c-47b9-a2ba-49197c4ca913)

![Screenshot_20240322-105949.png](https://github.com/community/community/assets/57124166/d0989151-aadd-4c02-993c-92d563d5af58)

**Reply by hpsin** [reply]
Thanks! We think we realized the issue here. That 28 day checkup lands you on this page _if_ you choose to reconfigure. At which point the UI assumes you _can't_ perform 2FA and that your access will be limited if you don't setup valid 2FA creds. So the language is misleading for your scenario. 


To validate, did you see a UI like this before hitting this banner? 
![image](https://github.com/community/community/assets/1666363/26f73956-6bc6-4fb5-a5fc-49def8f884ce)

**Reply by bearbones** [reply]
I don't remember anymore, I just remember tapping on the link text "click here to get started, screenshot of the URL shown by tap-and-hold 
![Screenshot_20240323-143640.png](https://github.com/community/community/assets/57124166/0a31573e-b5fa-41e1-acd4-71ea6c131e57)
